client/src/components/CustomTables.js

```diff
@@ -173,6 +174,28 @@ const upcomingOrOngoingTableStyles = makeStyles({
 export function UpcomingOrOngoingTable(props) {
   const classes = upcomingOrOngoingTableStyles();
   const [ongoingInterviewList, setOngoingInterviewList] = useState([]);
+  const [newongoingInterviewList, setnewongoingInterviewList] = useState([]);
+  function cancelInterview(id) {
+    console.log(id);
+    fetch("interviews/remove", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ id: id }),
+    })
+      .then((res) => console.log(res))
+      .then(() => {
+        let arr = ongoingInterviewList;
+        let index = arr.indexOf(id);
+        console.log(index);
+        if (index !== -1) {
+          console.log(ongoingInterviewList);
+          arr.splice(index, 1);
+          setOngoingInterviewList(arr);
+          window.location.reload();
```

Comment:
we shouldn't be using window.location.reload - by calling setOngoingInterviewList, it should cause the component to re-render, and if it is not, then we have an issue.